app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    ls = command.split()
```

Comment:
You should check if the input string contains exactly three elements and if the first element is 'cp', as specified in the checklist. This ensures that the command is correctly formatted.